ci_scripts/run_examples.sh

```diff
@@ -1,74 +0,0 @@
-cd examples
```

Comment:
Hey, what happened to this script? Are we still testing all the examples?

Reply:
Yes! While making the documentation:https://github.com/franchuterivera/SMAC3/runs/1681990495?check_suite_focus=true

But not, if we have a shell script. I missed this files and would add a Github action for this.